ertviz/models/plot_model.py

```diff
@@ -3,6 +3,41 @@
 import plotly.figure_factory as ff
 
 
+class BoxPlotModel:
+    def __init__(self, **kwargs):
+        self.selected = True
+        self._x_axis = kwargs["x_axis"]
+        self._y_axis = kwargs["y_axis"]
+        self._text = kwargs["text"]
+        self._name = kwargs["name"]
+        self._color = kwargs["color"]
+
+    @property
+    def repr(self):
+        repr_dict = dict(
+            # x=self._x_axis,
```

Comment:
remove

Reply:
I haven't removed it but made a comment that sort of clarifies what's needed to be done